pkg/db/db.go

```diff
@@ -256,6 +256,13 @@ func closeRows(rows *sql.Rows) error {
 	return nil
 }
 
+func closeRowsAndLog(rows *sql.Rows, ctx context.Context, prefix string) {
+	err := rows.Close()
+	if err != nil {
```

Comment:
some of the replaced code also did a null check:
```suggestion
    if rows != nil {
        return
    }
	err := rows.Close()
	if err != nil {
```